src/users/dto/userGetDto.ts

```diff
@@ -4,6 +4,11 @@ export const UserGetDto = (user: User): User => ({
   id: user.id,
   firstName: user.firstName,
   lastName: user.lastName,
+  password: user.email,
```

Comment:
```suggestion
  password: user.email,
```
Doesn't making sense, how password is equal to email?

Reply:
done